src/components/Calculator.js

```diff
@@ -0,0 +1,32 @@
+import './Calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <input type="text" className="calculator-screen" value={0} disabled />
+      <div className="calculator-keys">
+        <button type="button" className="all-clear" value="all-clear">AC</button>
+        <button type="button" className="plus-minus" value="plus-minus">+/-</button>
+        <button type="button" className="percent" value="percent">%</button>
+        <button type="button" className="operator" value="+">+</button>
+        <button type="button" value="7">7</button>
+        <button type="button" value="8">8</button>
+        <button type="button" value="9">9</button>
+        <button type="button" className="operator" value="*">&times;</button>
+        <button type="button" value="4">4</button>
+        <button type="button" value="5">5</button>
+        <button type="button" value="6">6</button>
+        <button type="button" className="operator" value="-">-</button>
+        <button type="button" value="1">1</button>
+        <button type="button" value="2">2</button>
+        <button type="button" value="3">3</button>
+        <button type="button" className="operator" value="/">&divide;</button>
+        <button type="button" className="zero" value="0">0</button>
+        <button type="button" className="decimal" value=".">.</button>
+        <button type="button" className="equal-sign" value="=">=</button>
+      </div>
+    </div>
+  );
```

Comment:
Good job on your design so far, but let's try to follow the design guidelines as this will help us in our future projects when working with clients or our team leader in the workforce.

- [ ] Your design does not match the required design given in the project requirement. It is required you match your design with the original design. 

<img width="921" alt="Screenshot 2023-04-27 at 5 40 52 PM" src="https://user-images.githubusercontent.com/50941074/234947805-0e2e607b-e33f-473c-b61d-9545de2e197d.png">


**Your Design**

<img width="424" alt="Screenshot 2023-04-27 at 5 40 04 PM" src="https://user-images.githubusercontent.com/50941074/234947558-0bab7449-9191-41cc-a82e-beb7421544cb.png">

**The Original Design**

<img width="312" alt="Screenshot 2023-04-27 at 5 40 21 PM" src="https://user-images.githubusercontent.com/50941074/234947734-9a6110a4-0cd2-476a-886c-26bfeff1e69b.png">

Reply:
Done